libass/ass.c

```diff
@@ -77,6 +77,22 @@ struct parser_priv {
     uint32_t sinfo_flags;
 };
 
+static const char *ass_style_format =
+        "Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, "
+        "OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, "
+        "ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, "
+        "Alignment, MarginL, MarginR, MarginV, Encoding";
+static const char *ass_event_format =
+        "Layer, Start, End, Style, Name, "
```

Comment:
`Actor` *(used for the entire git history) changed to `Name`.
As I can't find any other reference to "Actor", this was probably a typo, but if there is some variant of "standard format lines" I'm not aware of, which use `Actor` instead of `Name`, please let me know.